docker-qgis/entrypoint.py

```diff
@@ -172,6 +172,7 @@ def cmd_package_project(args: argparse.Namespace):
                 id="start_qgis_app",
                 name="Start QGIS Application",
                 method=qfc_worker.utils.start_app,
+                return_names=["qgis_app", "qgis_app_version"],
```

Comment:
Ideally we should never pass complex objects between the steps in the workflow. I think we can make `start_app` to return just the qgis version and nothing else. We use the global  `QGISAPP` anyways.

```suggestion
                return_names=["qgis_version"],
```

Reply:
Noted, thanks.